messenger.rst

```diff
@@ -209,6 +220,35 @@ This will also configure the following services for you:
 
         $ composer require symfony/amqp-pack
 
+Redis
+~~~~~
+
+The redis transport will use the redis streams to queue messages.
+
+.. code-block:: bash
+
+    # .env
+    ###> symfony/messenger ###
```

Comment:
To me this to flex specific. I would only focus on the ENV var (and the filepath)

Reply:
Thank you!
Think the same. Did remove it also from the amqp configuration example.